runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -293,19 +293,16 @@ message WorkItemCommitRequest {
   optional SourceState source_state_updates = 12;
   optional int64 source_watermark = 13 [default=-0x8000000000000000];
   optional int64 source_backlog_bytes = 17 [default=-1];
-  optional int64 source_bytes_processed = 22 [default = 0];
+  optional int64 source_bytes_processed = 22;
 
   repeated WatermarkHold watermark_holds = 14;
 
-  repeated int64 finalize_ids = 19 [packed = true];
-
-  optional int64 testonly_fake_clock_time_usec = 23;
-
   // DEPRECATED
   repeated GlobalDataId global_data_id_requests = 9;
 
   reserved 6;
```

Comment:
Nit but I think you can do
reserved 6, 19, 23;

Reply:
Ok, I'll make the change